lib/code_corps/messages/conversation_parts.ex

```diff
@@ -24,9 +25,14 @@ defmodule CodeCorps.Messages.ConversationParts do
   @spec create_changeset(ConversationPart.t, map) :: Ecto.Changeset.t
   def create_changeset(%ConversationPart{} = conversation_part, attrs) do
     conversation_part
-    |> cast(attrs, [:author_id, :body, :conversation_id])
+    |> cast(attrs, [:author_id, :body, :conversation_id, :part_type])
```

Comment:
I'm guessing this will need an update cast as well?  Don't see an update action however.

Reply:
I don't think we would cast this in an `update`, since the part type should never change.